lib/runner.js

```diff
@@ -54,7 +54,14 @@ class Runner extends Emittery {
 		let hasStarted = false;
 		let scheduledStart = false;
 		const meta = Object.freeze({
-			file: options.file
+			file: this.file,
+			get snapshotDirectory() {
+				return snapshotManager.determineSnapshotDir({
+					file: this.file,
+					fixedLocation: this.snapshotDir,
+					projectDir: this.projectDir
```

Comment:
`this` is probably `meta`, not the runner.

Reply:
Ah, yes, you’re probably right. The lambda took care of that :p

Reply:
Correct. This was the issue.